static/extensions/pooiod/Dictation.js

```diff
@@ -0,0 +1,88 @@
+// Dictation v1.3 by pooiod7
+
+(function(Scratch) {
+  'use strict';
+
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error('Your microphone can\'t be accessed in the sandbox');
+  }
+
+  class dictation {
+    constructor() {
+      this.recognizedSpeech;
+      this.cando = Scratch.vm.runtime.isPackaged;
```

Comment:
Dont use `runtime.isPackaged` in this use.

Use `typeof scaffolding === "undefined"`. It returns true if its not packaged

Reply:
why do that? isPackaged should just be yes if packaged runtime no if unpackaged?

Reply:
packaged runtime can be disabled in the packager (its very confusing)

Reply:
> why do that? isPackaged should just be yes if packaged runtime no if unpackaged?

Long story short, isPackaged is kind of misinformation. It doesn't report true if it's in a packaged environment. It returns true depending on certain packager information. 

For example, if you disable the "Remove Assets for RAM" option, isPackaged will return false